src/App.css

```diff
@@ -1,7 +1,268 @@
-*body {
-  background-color: #fff;
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  background-color: #f5f6fa;
+}
+
+#root {
+  margin: 5% 4%;
+  display: flex;
+  background-color: #fafafa;
+  box-shadow: 0 0 60px 0 rgb(0 0 0 / 5%);
 }
 
 .app {
-  margin-left: 20px;
+  width: 100%;
+}
+
+.app nav {
+  width: 100%;
+  background: white;
+  border-bottom: 1px solid #dfdfdf;
+  height: 85px;
+  margin-top: -14px;
+}
+
+.app ul {
+  display: flex;
+  gap: 32px;
+  list-style-type: none;
+  align-items: center;
+  padding: 0 2%;
+  padding-top: 18px;
+  margin-left: 30px;
+}
+
+.img {
+  display: flex;
+  justify-content: flex-end;
+  width: 64%;
+  margin-left: 60px;
+}
+
+.img img {
+  border: 1px solid #e8e8e8;
+  border-radius: 34px;
+  cursor: pointer;
+  padding: 8px;
+  width: 44px;
+}
+
+/* stylelint-disable selector-class-pattern */
+.headerName {
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+  padding-left: 5%;
+}
+
+.header-ul a {
+  opacity: 1;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  color: #121212;
+  letter-spacing: 1.9px;
+  text-decoration: none;
+}
```

Comment:
- [ ] In this stylesheet, I noticed that you repeated a lot of style declarations, especially for the **_background colours_**, **_font colours_**, `display: flex` rule, **_font-families_** and some other rules. I would advise you to organize each one as a class selector and use the class selector in the element that you desire. 👍 

### Hint: Let's say you have `h1` and `h2` tags that share similar style declarations:

```jsx
<h1 className="main-text">Hello World</h1>
<h2 className="smaller-text">Hello Micronaut</h2>
```

Instead of doing this:

```CSS
.main-text {
  font-family: "Montserrat", sans-serif;
  color: ##121212;
  background-color: #e8e8e8
}
.smaller-text {
  font-family: "Montserrat", sans-serif;
  color: ##121212;
  background-color: #e8e8e8
}
```

#### You could do this:

```CSS
.montserrat {
  font-family: 'Montserrat', sans-serif;
}

.dark-color {
  color: #121212;
}

.background-color {
  background-color: #e8e8e8;
}

.main-text {
  /* Specific style rule for the h1 tag */
}

.smaller-text {
  /* Specific style rule for the h2 tag */
}
```

#### Then apply it to the elements like this:

```jsx
  <h1 className="main-text montserrat dark-color background-color">Hello World</h1>
  <h2 className="main-text montserrat dark-color background-color">Hello World</h2>
```

### This way, you can use the syle rules across different elements without repeating them in the style sheets. It also helps in the maintainability and readability of code. 👍 😎